cookiecutter/prompt.py

```diff
@@ -1,7 +1,9 @@
 """Functions for prompting the user for project info."""
 import json
+import re
 from collections import OrderedDict
 
+from pathlib import Path
```

Comment:
`pathlib` is not a third party import, so it shouldn't be grouped with `rich` and `jinja2`.

Let me know if you're open to adding `isort` as a pre-commit hook to standardize this!